spacy/ml/models/parser.py

```diff
@@ -12,7 +12,8 @@
 @registry.architectures.register("spacy.TransitionBasedParser.v1")
 def build_tb_parser_model(
     tok2vec: Model[List[Doc], List[Floats2d]],
-    nr_feature_tokens: Literal[3, 6, 8, 13],
+    state_type: str,
```

Comment:
Now that we have the `Literal` type set up, we could consider making this `Literal["parser", "ner"]` 🤔